sdk/trace/span.go

```diff
@@ -277,6 +287,9 @@ func (s *recordingSpan) addOverCapAttrs(limit int, attrs []attribute.KeyValue) {
 
 	// Now that s.attributes is deduplicated, adding unique attributes up to
 	// the capacity of s will not over allocate s.attributes.
+	if limit-len(s.attributes) > 0 {
```

Comment:
Should it not be the following?

```suggestion
	if len(s.attributes) > limit {
```

Do we have any benchmark that shows this improvement? How about setting a limit in `BenchmarkRecordingSpanSetAttributes`?

Reply:
Oh yes, nice catch on the sense change. I will add the benchmark.